manifests/candlepin.pp

```diff
@@ -171,10 +173,12 @@
       certificate   => $ca_cert,
     }
 
+    include certs::foreman
     truststore_certificate { "${truststore}:${artemis_alias}":
       ensure        => present,
       password_file => $truststore_password_path,
-      certificate   => $client_cert,
+      certificate   => $certs::foreman::client_cert,
+      require       => File[$certs::foreman::client_cert],
```

Comment:
Don't we have some autorequire for this?
https://github.com/theforeman/puppet-certs/blob/e8d0346188b317c5ba354b92fc25fdb3a9285adc/lib/puppet/type/truststore_certificate.rb#L40-L42
```suggestion
```